tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/TableName.java

```diff
@@ -26,13 +26,20 @@
     private final Optional<String> schema;
     private final String name;
     private final String nameInDatabase;
+    private final boolean shouldCreateSchema;
```

Comment:
remove it from here, TableName is about existing tables in database, it has nothing to do with table creation